### PR TITLE
updated to include a reference to VS Code

### PR DESCRIPTION
**Pull request by DonJayamanne**
[Visual Studio Code](https://code.visualstudio.com) is a very lightweight, free and cross platform IDE.
This is MIT licensed (open source).

The [Python extension[(https://marketplace.visualstudio.com/items?itemName=donjayamanne.python) turns this IDE into an excellent Python Development Environment.
This too is open sourced and MIT licensed.

Features include (for python development) include:
- Integrated GIT
- Tasks
- Local and remote debugging
- Linting (multiple linters)
- Intellisense (signatures, auto completion, etc)
- Code navigation (find references, go to def, etc)
etc

### CODE REVIEW

File: docs/dev/env.rst [hunk 1]
```diff
@@ -150,6 +150,14 @@ features can be brought to IntelliJ with the free
 versions of PyCharm: Professional Edition (Free 30-day trial) and Community
 Edition(Apache 2.0 License) with fewer features.
 
+Python (on Visual Studio Code)
+-----------------------------
+
+`Python for Visual Studio <https://marketplace.visualstudio.com/items?itemName=donjayamanne.python>`_ is an extension for the `Visual Studio Code IDE <https://code.visualstudio.com>`_.
+This is a free, light weight and open source IDE, with support for Mac, Windows and Linux.
```
Comment (kennethreitz) [inline docs/dev/env.rst]: Oxford comma!

File: docs/dev/env.rst [hunk 2]
```diff
@@ -150,6 +150,14 @@ features can be brought to IntelliJ with the free
...
+Built using open source technologies such as NodeJs and Python, with compelling features such as Intellisense (autocompletion), local and remote debugging, linting, and the like.
```
Comment (kennethreitz) [inline docs/dev/env.rst]: Node.js

File: docs/dev/env.rst [hunk 3]
```diff
@@ -150,6 +150,14 @@ features can be brought to IntelliJ with the free
...
+MIT licensed.
```
Comment (kennethreitz) [inline docs/dev/env.rst]: Add a newline before this :)

File: docs/dev/env.rst (hunk 1 above)
Comment (DonJayamanne) [inline docs/dev/env.rst]: Would changing it to the following work?
>This is a free, light weight and open source IDE, with support for Mac, Windows and Linux.

**Comment by DonJayamanne** [comment]
Please re-review, changes have been made

File: docs/dev/env.rst (hunk 1 above)
Comment (kennethreitz) [inline docs/dev/env.rst]: yep!

**Comment by kennethreitz** [comment]
May need to revise whitespace still, but will merge and see how it renders!

Thanks, VS Code is a good project.

**Comment by DonJayamanne** [comment]
Thanks

**Pull request merged**